Open Judge System/Web/OJS.Web/Scripts/Administration/Problems/problems-create.js

```diff
@@ -48,27 +77,40 @@ $(document).ready(function () {
         numericTextBox.enable(true);
         input.attr("data-val-required", "Лимита е задължителен!");
 
-        $("form").removeData("validator");
-        $("form").removeData("unobtrusiveValidation");
-        $.validator.unobtrusive.parse($('form'));
+        $("#create-form form").removeData("validator");
+        $("#create-form form").removeData("unobtrusiveValidation");
+        $.validator.unobtrusive.parse($('#create-form form'));
```

Comment:
Please be consistent with the type of quotes in JavaScript. Use only `'` or only `"`, but not both.